start-service.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -xe
+source /home/ec2-user/.bash_profile
+cd /home/ec2-user/app/release
+nohup sudo bash create-swap.sh  > ../logs/run_swap.out 2> ../logs/run_swap.err &
```

Comment:
does it make sense to perform this `create-swap` in the launch template for the ec2 instance or is it connected to starting up the app?

Reply:
As in it getting setup as part of the Infrastructure Deployment? i.e. in deploy-infra.sh /main.yml?

I hadn't thought of doing it there, primarily because this was somewhat straightforward to implement. However, yea it would make sense to do this part as part of the instance initialization rather than on "app startup"